.github/CONTRIBUTING.md

```diff
@@ -2,81 +2,97 @@
 
 ## Our contribution model
 
-Carbon Design System is an **open source** project at IBM. We pride ourselves in
-open and inclusive design and development. If you're wondering more about our
-contribution process, you're in the right place. First off, thank you for your
-interest! This project is made possible not just by the core Carbon team, but
-also by several community members who have invested their own time to give back
-to the Carbon community.
+Carbon Design System is an **open source** project at IBM. We pride ourselves on
+open and inclusive design and development. If you're curious about our
+contribution process, you're in the right place. Thank you for your interest!
+
+This project is made possible not just by the core Carbon team, but by community
+members who’ve generously contributed their time to give back.
 
 ## Code of conduct
 
-We value all of our community members, and thus want to foster a positive
-contributing environment. Please take a look at our
-[code of conduct](./CODE_OF_CONDUCT.md) before engaging in our workspaces.
+<!--
+TODO:
+- Can this section be deleted? If not, can it be simplified to just a link?
+- Should the Code of Conduct be linked from the README? Seems like everyone,
+  including contributors, should see it there before they get to the
+  contributing docs. If so, having it in the README could be another reason to
+  delete this section.
```

Comment:
Simplyfing to just a link is fine, at the time this was penned I don't think github was surfacing the COC through the UI like it does now. I don't think we need it in the readme since it's already on the code tab.

![2025-04-21 at 15 40 44-carbon-design-systemcarbon A design system built by IBM-Google Chrome@2x](https://github.com/user-attachments/assets/05569d2d-1bcb-4f84-ad47-8becc8a7105d)